libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/internal/navigation/HttpClient.kt

```diff
@@ -0,0 +1,49 @@
+package com.mapbox.services.android.navigation.v5.internal.navigation
+
+import com.mapbox.navigator.HttpCode
+import com.mapbox.navigator.HttpInterface
+import com.mapbox.navigator.HttpResponse
+import java.io.ByteArrayOutputStream
+import okhttp3.OkHttpClient
+import okhttp3.Request
+import okio.Okio
+
+class HttpClient(
+    private val userAgent: String = "",
+    private val acceptGzipEncoding: Boolean = false
+) : HttpInterface() {
+
+    private val client = OkHttpClient
+        .Builder()
+        .build()
+
+    override fun isGzipped(): Boolean {
+        return acceptGzipEncoding
+    }
+
+    override fun get(url: String): HttpResponse {
+        val requestBuilder = Request.Builder()
+            .addHeader("User-Agent", userAgent)
```

Comment:
may be better use _null_ `userAgent` and don't hide issue under default value?

Reply:
removed default param. Now we have to always pass it to build HttpClient